packages/cli/lib/project-generator.js

```diff
@@ -55,6 +56,24 @@ module.exports = class ProjectGenerator extends BaseGenerator {
       const isValid = utils.validate(this.args[0]);
       if (typeof isValid === 'string') throw new Error(isValid);
     }
+
+    this.setupRenameTransformer();
```

Comment:
Is this line of code needed? I thought functions defined in Yeoman index files get called in sequential order they're defined in.

Reply:
Yep. If I comment it out and run CLI, the transformer is not registered ...